### ISSUE TITLE
嗨！这里有个外国人。需要帮助。

### ISSUE THREAD

**Issue by student1345** [ihmily/QuarkPanTool#13]
Hi! A non-Chinese person here.
Everytime I try to download something it sends this same thing & do not download anything in my Computer. Am I doing something wrong?

嗨！我是外国人。首先，我不懂中文。这只是翻译。
每次我尝试下载某些东西时，它都会发送相同的内容，并且不会在我的电脑中下载任何东西。我做错了什么吗？

"**[2024-09-26 23:13:44] 用户名：ak
[2024-09-26 23:13:44] 你当前选择的网盘保存目录: 根目录 文件夹
请输入你的选择(1—5或q)：1
请输入夸克文件分享地址：https://pan.quark.cn/s/313d829611dc
[2024-09-26 23:13:53] 文件分享链接：https://pan.quark.cn/s/313d829611dc
[2024-09-26 23:13:58] 转存总数：1，文件数：0，文件夹数：1
[2024-09-26 23:13:58] 文件转存列表：[]
[2024-09-26 23:13:58] 文件夹转存列表：['W无尽神域']
[2024-09-26 23:14:00] 获取任务ID：324140fa73ed4bc3a22dc315980b3cef
[2024-09-26 23:14:01] 第1次提交任务
[2024-09-26 23:14:04] 结束任务ID：324140fa73ed4bc3a22dc315980b3cef
[2024-09-26 23:14:04] 文件保存位置： 根目录 文件夹**"

**Comment by ihmily** [comment]
Hey guy, if you need download files to your computer, you should use option 5.
The option 1 means that save the files of other users to your quark clound, not your local computer storage 
But the tool only supports download files below 100M, it is the official webpage restriction of Quark Cloud,
If you continue to do so, it will prompt 'Download file too large. To ensure stable and secure download transmission, please use PC client to download'

如果你需要将文件下载到你的计算机上，你应该使用选项5。
选项1意味着将其他用户的文件保存到您的夸克克隆中，而不是您的本地计算机存储中
但该工具仅支持100M以下的下载文件，这是Quark Cloud的官方网页限制，
如果您继续这样做，它将提示“下载文件太大。为确保稳定安全的下载传输，请使用PC客户端下载”

**Comment by student1345** [comment]
Actually, there is a Telegram group which uploads video on quark and shares a cloud link, for example " https://pan.quark.cn/s/cc6d56974c5a ". If you click on this it will take you to a folder with some video files. I want to download just a single video file from here into my device. (The file is 1GB+).

I saved the video file into my own cloud account directly through the Quark Pan app. Also saved the whole folder. Also saved through your .exe. And paste the link in the  url.txt file. 

But still I can't download it into my device. 
It says, "[2024-09-30 18:50:17] The downloaded file must be a file in the network disk. Please transfer the file to the network disk first."

其实有个 Telegram 群在 quark 上上传视频，并分享一个云链接，比如“https://pan.quark.cn/s/cc6d56974c5a”。点击后会进入一个包含视频文件的文件夹。我想从这里下载一个视频文件到我的设备里。（文件 1GB+）。

我直接通过 Quark Pan app 将视频文件保存到我自己的云账户里。也保存了整个文件夹。也通过你的 .exe 保存。并将链接粘贴到 url.txt 文件中。

但还是无法下载到我的设备里。
提示 "[2024-09-30 18:50:17] 下载文件必须是网盘内文件，请先将文件转存至网盘中"

**Comment by student1345** [comment]
Yeah, I am already using the PC app, the issue is download speed is too slow. Max 500kb/s rarely, on average 300kb/s.
Anyway, thanks.
